src/toil/jobStores/abstractJobStore.py

```diff
@@ -903,6 +904,24 @@ def writeFileStream(self, jobStoreID=None, cleanup=False, basename=None):
                file basename so that when searching the job store with a query
                matching that basename, the file will be detected.
 
+        :param str mode: an optional string that specifies the mode in which the file is opened.
+                It defaults to 'b' which means open for writing in binary mode.
+
+                Currently supported mode characters:
+
+                    - 'b': open for writing in binary mode
+
+                    - 't': open for writing in text mode
+
+                The default mode is 'b' (open for writing in binary mode, synonym of 'wb').
```

Comment:
What does this mean if there's no `'w'`?

Reply:
I was thinking that 'w' is implied for both modes in `writeFileStream()` so I wrote the default as 'wb'. Would you recommend leaving it as 'b'  instead to avoid any confusion?